js/mapData.js

```diff
@@ -1,149 +1,128 @@
 var map;
 
 (function() {
-  'use strict';
-
-  var API_KEY = 'AIzaSyDziLuMxVaWGuE4BVh-gxvuY9y7evusUx0',
-      RANGE = 'A2:I',
-      SPREADSHEET_ID = '11W6MYoIIJDTJz0LL_P9WzHqT-EXpv9Bih6dlFLdAnfA',
-      publicEndpoint = 'https://sheets.googleapis.com/v4/spreadsheets/' + SPREADSHEET_ID + '/values/' + RANGE + '?key=' + API_KEY;
-  var const_API_KEY_MAPS = 'AIzaSyCtHdTVnmaA1ZQ-1wdaKoVEBtlTT020yXQ',
-      const_API_URL = 'https://maps.googleapis.com/maps/api/js?key=' + const_API_KEY_MAPS + '&callback=mapCallback';
-
-  function getData() {
-    fetch(publicEndpoint).then(function (result) {
-		return result.json();
+	'use strict';
```

Comment:
please see the indentation on how it was before. We are using 2 spaces per level right now, would be great if you could do the same, so we have consistent formatting of the code.